src/_includes/layouts/base.njk

```diff
@@ -13,6 +13,7 @@
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap" rel="stylesheet">
```

Comment:
We can load both font families in the same link from Google.

Reply:
I could not get that to work when I tried to do it this morning. Formatting the link the way Google suggested to load more than one font would produce an error and not load any of the fonts.

Reply:
Oh weird, ok ignore that comment for now then and we can check later what's going on.